Dockerfile

```diff
@@ -1,11 +1,13 @@
-FROM golang:1.16.15@sha256:35fa3cfd4ec01a520f6986535d8f70a5eeef2d40fb8019ff626da24989bdd4f1
+FROM --platform=${BUILDPLATFORM} golang:1.19.3@sha256:d388153691a825844ebb3586dd04d1c60a2215522cc445701424205dffc8a83e
+
+ARG TARGETOS TARGETARCH
```

Comment:
I _think_ both formats are supported, but defining multiple build-args in a single `ARG` can sometimes be somewhat ambiguous; the `ENV` instruction supports `ENV NAME VALUE` (without `=` in between). To prevent confusion, it may be clearer (although slightly more verbose) to use two separate `ARG` instructions;

```dockerfile
ARG TARGETOS
ARG TARGETARCH
```

Reply:
Agreed. Will update.